src/config_manager.sh

```diff
@@ -0,0 +1,69 @@
+declare -A paths
+declare -A comments
+
+DB_PATH=$DEFAULT_CONFIG_PATH/configdb
+
+function load_configs()
+{
+  local i=0
+
+  while read line
+  do
+    if echo $line | grep -F = &>/dev/null
+    then
+      i=$(($i+1))
+      paths[$i]=$(echo $line | cut -d '=' -f 1 | tr -d '[:space:]')
+      comments[$i]=$(echo "$line" | cut -d '=' -f 2-)
+    fi
+  done < $DB_PATH
+}
+
+function register_config()
```

Comment:
This feature is close to become nice... however, take a look in the following scenario:
```
cd my_kernel_branch/
kw rc .config "This is my working version of lala"

cd ../another_kernel_branch/
kw gc 1
```

This will not work well because you only save the path. The `kw` saving only the name of the config file. This feature will be correct if it saves the config file registered in the directory `config_files` with a name `config_<INDEX>`; then, the `kw gc` just take a copy of it in the current directory. With this change, we will have a real backup system of `.config` files.

Finally, it could be nice if we can have an alias option for each config file that we manager instead of using the index number. For example:
```
kw rc .config "This config file handle my Rasp" rasp
kw rc .config "This config file handle my x86 pc" x86host
kw sc
1) rasp "This config file handle my Rasp"
2) x86host "This config file handle my x86 pc"
kw gc rasp # or kw gc 1
kw gc x86host # or kw gc 2
```